sdk/metric/metricdata/data.go

```diff
@@ -126,6 +132,9 @@ type HistogramDataPoint struct {
 	Max Extrema
 	// Sum is the sum of the values recorded.
 	Sum float64
+
+	// Exemplars is the sampled Exemplars collected during the timeseries.
+	Exemplars []Exemplar[N] `json:",omitempty"`
```

Comment:
I think we should also change the types of `Bounds`, `Min`, `Max`, Sum` to respect the type parameter. Separate PR?

Reply:
Yeah, agreed, and yeah also separate PR. :+1: